MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -42593,33 +42629,10 @@
       "ConditionalItems": [
         [
           "Use Fire Arrow",
-          "MaskGoron"
-        ],
-        [
-          "Use Fire Arrow",
-          "1 Snowhead Key",
-          "SHT Zora Jumps"
```

Comment:
How come SHT Zora Jumps no longer required for this?

Reply:
Again, I wanted to simply add the jump to the 3rd locked door in logic by default, but thinking about it more, it could probably be changed to require something even if the jump can be done as human. I'll probably change this the next time I get around to casual logic, but I think it's fine for now.